app/components/Vault/index.js

```diff
@@ -597,18 +612,20 @@ const Vault = (props) => {
       ? new BigNumber(balanceOf)
           .dividedBy(10 ** decimals)
           .multipliedBy(pricePerShare / 10 ** decimals)
-          .toFixed()
+          .toFixed(vaultBalanceDecimalPlacesCount)
       : '0.00';
   } else if (vaultIsBackscratcher) {
     vaultBalanceOf = balanceOf
-      ? new BigNumber(balanceOf).dividedBy(10 ** decimals).toFixed()
+      ? new BigNumber(balanceOf)
+          .dividedBy(10 ** decimals)
+          .toFixed(vaultBalanceDecimalPlacesCount)
       : '0.00';
   } else {
     vaultBalanceOf = balanceOf
       ? new BigNumber(balanceOf)
           .dividedBy(10 ** decimals)
           .multipliedBy(getPricePerFullShare / 10 ** 18)
-          .toFixed()
+          .toFixed(vaultBalanceDecimalPlacesCount)
```

Comment:
We need to make sure to leave `vaultBalanceOf` in-tact. Currently this value is used to determine the maximum amount (in WEI) for a withdrawal. If we add decimals here I believe users will be left with trace amount of vault assets if they withdraw all.

The preference is to not alter `vaultBalanceOf` at all, instead only alter the output formatter of `AnimatedNumber` as you've done above.

Reply:
oops that'd be bad. Not sure why I had this code because it was already being formatted 🤔